client/dataEntry.js

```diff
@@ -31,20 +31,12 @@ Template.dataEntry.helpers({
   },
 
   isSelectedRoundClosed: function() {
-    if(!this.roundId) {
-      // If there's no round selected, then the selected round is definitely
-      // *not* closed =)
-      return false;
-    }
-    return Rounds.findOne(this.roundId).isClosed();
+    round = Rounds.findOne(this.roundId);
```

Comment:
I like the variant. I didn't realize these functions aren't intended to work without a roundId.

***

I like this idea of a sanity checker. That sounds like something that would help me sleep better at night, and allow me to be a less paranoid coder. :+1: to a Sanity Checker for Mongo or Meteor!